hoomd/md/external/wall.py

```diff
@@ -212,6 +212,20 @@ def walls(self, wall_list):
         if self._walls is wall_list:
             return
         self._walls = hoomd.wall._WallsMetaList(wall_list, _to_md_cpp_wall)
+        if self._attached:
```

Comment:
Does this fix a bug with MD walls?
If yes, we need to add add a draft change log entry on that fix to the PR description.

Reply:
Yes. It would have led to inconsistent wall lists between the python and c++ code if the walls setter was used to modify the walls in the simulation.